winterfell/services/userService.js

```diff
@@ -77,3 +79,74 @@ module.exports.upsertUserValues = function(userId, keyValues, callback) {
     userValue.save(callback);
   });
 };
+
+/**
+ * Expected input object from idme:
+ * {
+ *  id
+ *  verified
+ *  affiliation
+ *  fname
+ *  lname
+ *  zip
+ *  uuid
+ *  email
+ * }
+ */
+module.exports.createNewUserFromIdMe = function(idmeInfo, callback) {
+  var newUser = {
+    firstname: idmeInfo.fname,
+    middlename: null,
+    lastname: idmeInfo.lname,
+    email: idmeInfo.email,
+    password: "fakepassword", // todo: generate a short random 8-char password
```

Comment:
we will need a way to prevent them from entering email + password then.
one way could be to have a boolean flag that prevents them to. Or generate a random password that we never give to the user and it won't work if they try. When we generate a random password, if a user ever wants to go through email, they could always reset it.